app/example/src/main/java/com/cloudera/oryx/example/ExampleBatchLayerUpdate.java

```diff
@@ -15,12 +15,13 @@
 
 package com.cloudera.oryx.example;
 
-import com.cloudera.oryx.lambda.BatchLayerUpdate;
-import com.cloudera.oryx.lambda.TopicProducer;
+import java.io.IOException;
```

Comment:
My concern is that this also reordered all the imports. I think it was fairly consistent at, roughly:

java
third-party
project

I'd like to not change that everywhere as part of this change, but could make imports match that pattern along the way if we're already touching files' imports that are out of order.